easybuild/easyblocks/generic/pythonpackage.py

```diff
@@ -222,6 +223,10 @@ def __init__(self, *args, **kwargs):
 
         self.install_cmd_output = ''
 
+        if build_option('debug'):
+            global PIP_INSTALL_CMD
+            PIP_INSTALL_CMD += ' --verbose'
```

Comment:
@smoors I would move this below, where `self.install_cmd = PIP_INSTALL_CMD` is used, as:

```python
if build_option('debug'):
    self.install_cmd += ' --verbose'

Reply:
@boegel care to explain why you think that's better?